corehq/apps/export/models/new.py

```diff
@@ -68,8 +72,9 @@ class ExportColumn(DocumentSchema):
     item = SchemaProperty(ExportItem)
     label = StringProperty()
     # Determines whether or not to show the column in the UI Config without clicking advanced
-    show = BooleanProperty(default=False)
+    is_advanced = BooleanProperty(default=False)
     selected = BooleanProperty(default=False)
+    tags = ListProperty()
```

Comment:
not the best. `ExportItem` now has a `tag` property and `ExportColumn` has a `tags` property. the item has to have a tag in case that specific system property has an associated tag. more UI tags can be added later such as DELETED (but this is not conveyed in the exportitem). we may be able to condense to just have the ExportColumn use tags